pkg/api/common/const.go

```diff
@@ -59,7 +59,7 @@ const (
 
 const (
 	// KubernetesDefaultRelease is the default Kubernetes release
-	KubernetesDefaultRelease string = "1.10"
+	KubernetesDefaultRelease string = "1.11"
```

Comment:
this is almost too easy :p

Reply:
Lol, this is a not-so-simple change for me :P. Got caught off-guard